test/FunctionsV2/PlatformSpecificHelpers.FunctionsV2.cs

```diff
@@ -157,6 +215,21 @@ private static IWebJobsBuilder AddMultipleDurabilityProvidersDurableTask(this IW
             return builder;
         }
 
+        private static IWebJobsBuilder AddDurableClientFactoryDurableTask(this IWebJobsBuilder builder, IOptions<DurableTaskOptions> options, IEnumerable<IDurabilityProviderFactory> durabilityProviderFactories = null)
```

Comment:
Could we please add a short method-header comment to this?

Reply:
Is there anything in particular that you think would be helpful to add in the comment? I'm just asking since the other methods in TestHelpers.cs don't have comments.

Reply:
Hmm, it's hard to say, **I would be good with any blurb about when to invoke this method and for why it's needed**.
The reason why I ask is that this method is written at a very high level of abstraction (as it should due to interfacing w/ DI and factories) but that can make it difficult, for me, to know where and how to invoke it.

In short, I'm not too picky so any short comment about what this does and when to use it would do it for me :)